jablib/src/main/java/org/jabref/model/entry/field/FieldTextMapper.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.model.entry.field;
+
+import java.util.StringJoiner;
+
+import org.jabref.model.strings.StringUtil;
+
+public class FieldTextMapper {
+    public static String getDisplayName(Field field) {
+        if (field.isStandardField()) {
+            StandardField standardField = (StandardField) field;
+            return switch (standardField) {
```

Comment:
Nice start so that we could customize

`urldate` to `URL date` or even `Date of URL`.